src/lib/errorsHandler.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
```

Comment:
i've read somewhere about lib/libs directory for this kind of files, propably it was on stack.